pkg/kubectl/rolling_updater.go

```diff
@@ -139,36 +142,34 @@ func (r *RollingUpdater) Update(config *RollingUpdaterConfig) error {
 	}
 
 	// +1, -1 on oldRc, newRc until newRc has desired number of replicas or oldRc has 0 replicas
-	for newRc.Spec.Replicas < desired && oldRc.Spec.Replicas != 0 {
+	for newRc.Spec.Replicas < desired && (oldRc == nil || oldRc.Spec.Replicas != 0) {
 		newRc.Spec.Replicas += 1
-		oldRc.Spec.Replicas -= 1
-		fmt.Printf("At beginning of loop: %s replicas: %d, %s replicas: %d\n",
-			oldName, oldRc.Spec.Replicas,
-			newName, newRc.Spec.Replicas)
-		fmt.Fprintf(out, "Updating %s replicas: %d, %s replicas: %d\n",
-			oldName, oldRc.Spec.Replicas,
-			newName, newRc.Spec.Replicas)
+		if oldRc != nil {
+			oldRc.Spec.Replicas -= 1
+		}
+
+		fmt.Printf("At beginning of loop: %s\n", rollingLabelFor(newRc, oldRc))
```

Comment:
FWIW, there should be a way to turn off this output, for this and all other kubectl operations.